sdv/_utils.py

```diff
@@ -348,3 +348,83 @@ def check_sdv_versions_and_warn(synthesizer):
 
             message = f'{message} {static_message}'
             warnings.warn(message, SDVVersionWarning)
+
+
+def _check_greater_version(current_version, synthesizer_version):
```

Comment:
minor nitpick: can we swap the argument order here or change the function name to `_check_is_lower_version`? It's counterintuitive to me that `_is_greater_version(a, b)` returns `True` when `a < b` 😅